### PR TITLE
Main

### LINKED ISSUE #255: Add ids to contributor listing, so you can link to a specific one
## Is your feature request related to a problem? Please describe

It may be beneficial to provide direct links to your own contributor listing.

## Describe the solution you'd like

Every listing has an id, that can be jumped to. For that the github username seems to be the most sensible.

## Additional context

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_an_element_on_the_same_page

### PR DESCRIPTION
**Pull request by Asin-Junior-Honore**
# The Pull Request is ready

- [ x] fixes #255

### CODE REVIEW

File: website/src/pages/contributing/contributors/index.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ const Index = () => {
         <ExternalLink to={contributor.url} label={contributor.name} />
       </h2>
       <p>{contributor.bio}</p>
+      <a href={`#${contributor.id} `}>{contributor.name}</a>
```
Comment (Idrinth) [inline website/src/pages/contributing/contributors/index.tsx:33]: this is not the intended solution. you need something to link to, not a new link ;)

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: website/src/pages/contributing/contributors/index.tsx (hunk 1 above)
Comment (Asin-Junior-Honore) [reply website/src/pages/contributing/contributors/index.tsx:33]: @Idrinth alright could you please provide more information for me to well understand and know how to go about the issue please 🙏

Comment (Idrinth) [reply website/src/pages/contributing/contributors/index.tsx:33]: You need to create a link target, not a link. See the desired solution in the issue :)

Comment (Idrinth) [reply website/src/pages/contributing/contributors/index.tsx:33]: Hi @Asin-Junior-Honore I am assuming you gave up on the issue?

**Pull request closed without merging**